tests/tf-manifests/rhcs/clusters/rosa-classic/main.tf

```diff
@@ -45,6 +45,11 @@ locals {
     operator_role_prefix = var.operator_role_prefix
     oidc_config_id       = var.oidc_config_id
   }
+  base_dns_domain      = var.base_dns_domain == null ? null : var.base_dns_domain
+  private_hosted_zone  = var.private_hosted_zone_id == null ? null : {
```

Comment:
why not having this as a variable directly instead of constructing ?
This is not bringing more information in locals, you could use directly the var

Reply:
Good question.  Passing `private_hosted_zone_id` and `private_hosted_zone_role_arn` variables keep the same usage as OCM API:
```
```

Reply:
ok but in TF we have a specific struct for private hosted zone, to I expect this file here to use that form and have the test code adapt to it. wdyt ?

Reply:
Updated.